### PR TITLE
Feat/support new gnoscan

### PR DESCRIPTION
**Pull request by AkydA**
Feature
---
- support new version of gnoscan
- change env structure for network setting

### CODE REVIEW

File: packages/web/src/containers/token-description-container/TokenDescriptionContainer.tsx [hunk 1]
```diff
@@ -31,12 +32,13 @@ export const descriptionInit: DescriptionInfo = {
     website_url: "string",
   },
   links: {
-    Website: "gnoswap.io",
-    Gnoscan: "gnoscan.io/tokens/r_demo_grc20_GNOS",
+    Website: "https://gnoswap.io",
+    Gnoscan: "https://gnoscan.io/tokens/r/demo/wugnot",
```
Comment (r3v4s) [inline packages/web/src/containers/token-description-container/TokenDescriptionContainer.tsx:36]: data was GNOS, but WUGNOT ??

File: packages/web/src/hooks/common/use-gnoscan-url.tsx [hunk 2]
```diff
@@ -0,0 +1,64 @@
...
+  const network = useAtomValue(CommonState.network);
+
+  const getGnoscanUrl = (
+    type: GnoscanDataType | "" = "",
+    params = "",
+  ): string => {
+    const chainId = network.chainId || "";
+    const baseUrl = network.scannerUrl || "";
+    let chainParams = "";
+    if (["portal-loop", "test3"].includes(chainId)) {
```
Comment (r3v4s) [inline packages/web/src/hooks/common/use-gnoscan-url.tsx:25]: I thought test3 is deprecated

**Review by r3v4s — APPROVED** [review]
left question, but very minor thing

File: packages/web/src/containers/token-description-container/TokenDescriptionContainer.tsx (hunk 1 above)
Comment (AkydA) [reply packages/web/src/containers/token-description-container/TokenDescriptionContainer.tsx:36]: it's just dummy data

File: packages/web/src/hooks/common/use-gnoscan-url.tsx (hunk 2 above)
Comment (AkydA) [reply packages/web/src/hooks/common/use-gnoscan-url.tsx:25]: yepp that's true. but this check up condition is due to current gnoscan implement.

**Review by jinoosss — APPROVED** [review]

**Pull request merged**